packages/replay/src/bin.ts

```diff
@@ -38,6 +37,7 @@ program
     "--api-key <key>",
     "Authentication API Key"
   )
+  .option('--include-in-progress', 'Upload all recordings, including ones with an in progress status')
```

Comment:
I don't think we need this here because the user has already chosen the recording to upload so the status isn't relevant.